components/camel-chatscript/src/main/java/org/chatscript/Bot/ChatScriptBot.java

```diff
@@ -0,0 +1,76 @@
+package org.chatscript.Bot;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+import java.util.HashMap;
+
+import org.apache.camel.component.ChatScriptProducer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ChatScriptBot {
```

Comment:
What this class does? A large number of hardcoded values which are not clear.
The package must be start with org.apache.camel....

Reply:
Re-structured the packages. This class connects to chatscript server to push the message and get the reponse back.